go.mod

```diff
@@ -30,6 +29,8 @@ require (
 	google.golang.org/protobuf v1.30.0
 )
 
+require github.com/mattn/go-colorable v0.1.12 // indirect
```

Comment:
Why two requires? I think you can add it in the below lines

Reply:
I used `go mod tidy` 
I will add it to the first one.